src/main/java/core/basesyntax/model/Draw.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.model;
+
+public interface Draw {
+    double getArea();
+}
```

Comment:
The interface name 'Draw' is a verb, which is not recommended for class/interface names. Consider renaming it to something more indicative of its purpose, like 'Drawable' if its intended use is to ensure that a class has drawing capabilities. Additionally, the interface should not have a 'getArea' method, as drawing and calculating area are conceptually different. You might want to split this into two different interfaces, one for drawing and one for area calculation.